### ISSUE TITLE
spl token creation

### ISSUE THREAD

**Issue by ahmediblao1** [cryptoloutre/solana-tools#14]
❌ Ohoh.. failed to post funding tx - 5uDq2wKUvePWebVGHePfBozkpSLMNi5CDQ4fGQq2EaTK3BGPo55J9KYsAmqp96u8Dko23UPCZoBw1J1ubF9f44UY (keep this id!) - Confirmed tx not found

im facing this error i need help

**Comment by cryptoloutre** [comment]
@ahmediblao1 Hi, it's a known issue and it seems it comes from the current congestion and I don't know if I can do something. You can try to upload your token metadata separately with [this tool](https://solanatools.vercel.app/upload) to have an Arweave URI and then create your token with an existing URI. Instead to upload your token metadata to arweave, you can also store them in a Github file and use the permalink.

In both cases, the file has to look like this
````
{
  "name": "ZEUS",
  "symbol": "ZEUS",
  "description": "Cross-chain Communication Layer Built on SVM",
  "image": "https://raw.githubusercontent.com/ZeusNetworkHQ/zeus-metadata/master/logo-v2.png"
}
```

**Comment by ahmediblao1** [comment]
so as i understand i create json file with the token info then use the uri of Arweave

**Comment by ahmediblao1** [comment]
@cryptoloutre but it also when i want to upload the file it gives same error 
![Uploading SharedSllcreenshot.jpg…]()

**Comment by cryptoloutre** [comment]
@ahmediblao1 yes you have to create a json file first, upload it somewhere (Arweave, GitHub, etc) in order to have an URL and then you use this url in the tool to create your token

**Comment by ahmediblao1** [comment]
did that bit now on mainnet when i try to create gives error 
![image](https://github.com/cryptoloutre/solana-tools/assets/98063069/d0732405-eaef-4b2f-9d67-b29cd0551265)
        Failed to load resource: node1.bundlr.network/account/balance/solana:1 
 the server responded with a status of 400 ()
